src/passkit.cs

```diff
@@ -1140,7 +1178,11 @@ interface PKPaymentAuthorizationControllerDelegate {
 		[MacCatalyst (14,0)]
 		[Export ("paymentAuthorizationController:didRequestMerchantSessionUpdate:")]
 		void DidRequestMerchantSessionUpdate (PKPaymentAuthorizationController controller, Action<PKPaymentRequestMerchantSessionUpdate> handler);
-	
+
+		[NoWatch, Mac (12,0), iOS (15,0)]
```

Comment:
`[MacCatalyst (15,0)]` ?
otherwise it's implied it existed in 13.0 (the original release)